.browserslistrc

```diff
@@ -0,0 +1,3 @@
+# Browser Support
```

Comment:
Seems small but I like the comment in the docs for browserslist `# Browsers that we support`